templates/zerver/emails/missed_message.source.html

```diff
@@ -45,9 +45,10 @@
 
 
     <p>
-        <a href="{{ realm_uri }}">Click here to log in to Zulip and view your new messages.</a>
         {% if reply_to_zulip %}
-        Or just reply to this email.
+        Reply to this email directly, or <a href="{{ realm_uri }}">view it in Zulip</a>.
```

Comment:
@rishig I don't think we should say "view it in Zulip" for something that isn't a direct link to the conversation; it's misleading.

Reply:
oh, I didn't realize the links were not links to the conversation itself. How hard would that be to change, now that we don't commingle different narrows?

Reply:
We have nice functions for linking directly to the narrow (and in fact do so in the recipient headers in the previous design), so I don't think this should be a problem.